lib/commons/dom/is-visible.js

```diff
@@ -33,7 +33,9 @@ function isClipped(clip) {
  */
 dom.isVisible = function(el, screenReader, recursed) {
 	'use strict';
-	var style, nodeName, parent;
+	const node = axe.utils.getNodeFromTree(el);
+	const cacheName = 'isVisible' + (screenReader ? 'ScreenReader' : '');
+	let style, nodeName, parent, isVisible;
```

Comment:
Can these be moved to `const` and be declared when necessary. It will make it easier to follow, rather than declaring all of them here & initialising later.